### PR TITLE
Internationalization support

### PR DESCRIPTION
**Pull request by LeeAlephium**
Resolves https://github.com/alephium/desktop-wallet/issues/192

~~All that remains is extracting all the translated strings.~~

### CODE REVIEW

**Comment by mvaivre** [comment]
Excellent, thanks. It's good to have this ready, so we can make sure to always add the english base string when building new components.

With that being said, we'll wait before providing any translations : 

1) We need to make sure we have the bandwidth / community involvement first to maintain up to date translations.
2) There may be some political implications depending on the language(s) we provide first. We'll wait for direct input from our community to decide which languages we wanna add.

**Comment by LeeAlephium** [comment]
There's already an issue opened to determine the language(s) to provide first: https://github.com/alephium/desktop-wallet/issues/272

File: src/pages/Wallet/WalletLayout.tsx
Comment (nop33) [inline src/pages/Wallet/WalletLayout.tsx:144]: spaces

File: src/pages/WalletManagement/ImportWordsPage.tsx
Comment (nop33) [inline src/pages/WalletManagement/ImportWordsPage.tsx:58]: spaces

**Review by nop33 — COMMENTED** [review]
Hm, do we want to merge this as part of the upcoming wallet release? It won't essentially change anything until new language translations are added and we implement a language toggle, right?

/cc @LeeAlephium @mvaivre

File: src/index.tsx
Comment (mvaivre) [reply src/index.tsx:41]: Yes. `Suspense` accepts a react node as a fallback. We could use a spinner.

File: src/contexts/global.tsx
Comment (LeeAlephium) [reply src/contexts/global.tsx:189]: I think he means to follow code style? Sure, no problem

**Review by nop33 — APPROVED** [review]
Let's merge this after #244 is merged and we have rebased to `master`.

**Review by nop33 — APPROVED** [review]
Ready to be merged

**Comment by LeeAlephium** [comment]
:ok_hand: 

Thank you for the rebase!

**Pull request merged**
